core/src/main/java/org/apache/iceberg/stats/StatsCompressionCodec.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.stats;
+
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
+
+public enum StatsCompressionCodec {
+  /**
+   * LZ4 single compression frame with content size present
+   */
+  LZ4("lz4"),
+
+  /**
+   * Zstandard single compression frame with content size present
+   */
+  ZSTD("zstd"),
+  /**/;
```

Comment:
empty JavaDoc

Reply:
That's an empty block comment. That's a what we use in Trino to make formatter happy and to allow addition of new enum options without changing lines of existing options (clean git annotate output).